test/cli.js

```diff
@@ -364,6 +364,25 @@ test('bails when config contains `"tap": true` and `"watch": true`', t => {
 	});
 });
 
+['--concurrency', '-c'].forEach(concurrencyFlag => {
+	test(`bails when ${concurrencyFlag} provided without value`, t => {
+		execCli([concurrencyFlag, 'test.js'], {dirname: 'fixture/concurrency'}, (err, stdout, stderr) => {
+			t.is(err.code, 1);
+			t.match(stderr, 'The --concurrency and -c flags must be provided the maximum number of test files to run at once.');
+			t.end();
+		});
+	});
+});
+
+['--concurrency', '-c'].forEach(concurrencyFlag => {
```

Comment:
@novemberborn This tests for success if the value is provided for the flags. Looks good?